ds-prep-python-data-series/dict_columns.py

```diff
@@ -0,0 +1,130 @@
+# Data Set: Do Not Modify
+columns = ("year", "unemployment_rate")
+label_order = [
+    2001, 2002, 2003, 2004, 2005,
+    2006, 2007, 2008, 2009, 2010,
+    2011, 2012, 2013, 2014, 2015,
+    2016, 2017, 2018, 2019, 2020,
+    2021
+]
+unemployment_rates = {
+    'year': [
+        2009, 2011, 2005, 2016, 2001,
+        2012, 2019, 2007, 2021, 2018,
+        2010, 2013, 2006, 2017, 2015,
+        2014, 2008, 2004, 2003, 2020,
+        2002
+    ],
+    'unemployment_rate': [
+        8, 10, 6, 5, 5,
+        9, 4, 5, 7, 4,
+        11, 8, 5, 5, 6,
+        7, 5, 6, 6, 4,
+        6
+    ]
+}
+
+# Write your code below this line:
+
+print(unemployment_rates['year'][0])
+
+def last_series_point(columns, unemployment_rates):
+    return {columns[0]: unemployment_rates['year'][-1], columns[1]: unemployment_rates['unemployment_rate'][-1]}
+
+print('last_series_point: ', last_series_point(columns, unemployment_rates))
+
+
+def first_five_element_series(unemployment_rates):
+    first_five_tuples = []
+    for i in range(5):
+      first_five_tuples.append((unemployment_rates['year'][i], unemployment_rates['unemployment_rate'][i]))
+    return first_five_tuples
+
+print('first_five_element_series: ', first_five_element_series(unemployment_rates))
+
+def check_include_2000(unemployment_rates):
+    res_2000 = False
+    for i in unemployment_rates['year']:
+        if 2000 == i:
+            res_2000 = True
+    return res_2000
+
+print('check_include_2000: ', check_include_2000(unemployment_rates))
+
+def check_include_2010(unemployment_rates):
+    res_2010 = False
+    for i in unemployment_rates['year']:
+        if 2010 == i:
+            res_2010 = True
+    return res_2010
+
+print('check_include_2010: ', check_include_2010(unemployment_rates))
```

Comment:
You probably don't need two separate function definitions here, since the the only difference is which year you're looking for.

Reply:
create one function to return both the values